exp/views/study.py

```diff
@@ -276,3 +300,10 @@ def get_context_data(self, **kwargs):
             }, indent=4) for resp in context['responses']]
         context['all_responses'] = ', '.join(context['response_data'])
         return context
+
+
+class PreviewProxyView(ProxyView, LoginRequiredMixin):
+    upstream = settings.EXPERIMENT_BASE_URL
+
+    def dispatch(self, request, path, *args, **kwargs):
```

Comment:
If you're not doing anything but calling super you can just remove this method and it will inherit from the parent.